src/03-modality-agnostic-files.md

```diff
@@ -27,6 +27,7 @@ Every dataset MUST include this file with the following fields:
 | EthicsApprovals    | OPTIONAL. List of ethics committee approvals of the research protocols and/or protocol identifiers.                                                                                                                                                                |
 | ReferencesAndLinks | OPTIONAL. List of references to publication that contain information on the dataset, or links.                                                                                                                                                                     |
 | DatasetDOI         | OPTIONAL. The Document Object Identifier of the dataset (not the corresponding paper).                                                                                                                                                                             |
+| HEDVersion         | RECOMMENDED. HED version number or the name of the HED XML schema file located in `sourcedata` used to validate HED tags for study.                                                                                                                                              |
```

Comment:
this looks good, but I would perhaps specify that the file name is supposed to be:
- separated with `/` (as opposed to `\` as sometimes (always?) used on Windows)
- relative to `sourcedata` (with leading `/` or not?)

Reply:
The file name would be just the file, without any path information. The code will construct the path to `sourcedata/` during validation in a platform-agnostic manner.

Reply:
this line now needs to be updated now to follow our most recent agreement that we'll only accept an HED version number in the BIDS spec right now.